Src/IronPython/Lib/iptest/file_util.py

```diff
@@ -76,155 +76,20 @@ def delete_files(self, *files):
             except: pass
 
     def clean_directory(self, path, remove=False):
+        import shutil
```

Comment:
I'm tempted to say we should just remove the `remove` argument (always remove the directory).